src/main/views/search.njk

```diff
@@ -1,48 +1,79 @@
+            <div class="no_padding govuk-grid-column-three-quarters">
+                {{ govukErrorSummary({
+                    titleText: "There is a problem",
+                    errorList: [
+                        {
+                            text: errorMessage,
+                            href: "search-input-container"
+                        }
+                    ]
+                }) }}
+            </div>
+        </div>
+    {% endif %}
+    {% set inputClasses = 'govuk-!-width-two-thirds govuk-input--error'
+        if noResultsError or invalidInputError else 'govuk-!-width-two-thirds' %}
+    {% set formClasses = 'govuk-form-group--error govuk-form-group govuk-!-width-one-half'
+        if noResultsError or invalidInputError else 'govuk-form-group govuk-!-width-one-half' %}
+    <form action="search" method="post">
+        <div class="{{ formClasses }}">
+            <h1 class="govuk-label-wrapper">
+                <label class="govuk-label govuk-label--l" for="search-input">
+                    What court or tribunal are you interested in?
+                </label>
+            </h1>
+            <div id="search-input-hint" class="govuk-hint">
+                For example Blackburn Family Court or Chester
+            </div>
+            {% if noResultsError or invalidInputError %}
+                <span class="govuk-error-message">Enter a court name, town or city</span>
+            {% endif %}
+            <div class="{{ inputClasses }}" id="search-input-container"></div>
+        </div>
+        {{ submitButton() }}
+        {% if noResultsError %}
+        <div class="govuk-grid-row">
+            <div class="no_padding govuk-grid-column-full">
+                <h2 class="govuk-heading-s">There are no matching results.</h2>
+                <p class="govuk-body">You can improve your results by double checking your spelling.</p>
+            </div>
+        </div>
+        {% endif %}
+    </form>
 {% endblock %}
-
 {% block bodyEnd %}
-  {% set optionList = [] %}
-  {% for item in autocompleteList %}
-    {% set _ = optionList.push(
-      item.name
-    ) %}
-  {% endfor %}
+    {% set optionList = [] %}
+    {% for item in autocompleteList %}
+        {% set _ = optionList.push(
+            item.name
+        ) %}
+    {% endfor %}
 
-  <script src="/assets/js/accessible-autocomplete.min.js"></script>
-  <script>
-    const element = document.querySelector('#search-input-container')
-    const source = {{ optionList | dump |safe }}
-    accessibleAutocomplete({
-      element: element,
-      id: 'search-input',
-      source: source
-    })
-  </script>
+    <script src="/assets/js/accessible-autocomplete.min.js"></script>
+    <script>
+      const element = document.querySelector('#search-input-container')
+      const source = {{ optionList | dump |safe }}
+        accessibleAutocomplete({
+          element: element,
+          id: 'search-input',
+          source: source,
+          autoselect: true
```

Comment:
do we still need this if were not restricting people to searching by court name?

Reply:
removed autoselect